docs/_documentations/remote-deploying-codewind.md

```diff
@@ -19,9 +19,9 @@ Follow the instructions to deploy Codewind remotely. After you install your loca
 1. Install the Codewind operator in your cloud.
 2. Deploy your Codewind instances. 
 
-**Note: To learn how to connect your local IDE to your remote deployment of Codewind once it has been deployed remotely** see [Using Codewind remotely](remote-codewind-overview.html).
+**Note: To learn how to connect your local IDE to your remote deployment of Codewind after it has been deployed remotely**, see [Using Codewind remotely](remote-codewind-overview.html).
```

Comment:
The style guide mentions formatting notes like **Note:**. I think it would be good to either revisit the style guide and expand our use of bold or make this Note match others.